doc/changelog/05-tactic-language/13232-ltac2-if-then-else.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  Added an if-then-else syntax to Ltac2
```

Comment:
```suggestion
  Add an if-then-else construct to Ltac2
```

Reply:
Or even remove the "Add" part since it's already present in bold:
```suggestion
  An if-then-else syntax to Ltac2
```